src/extensions/default/RelatedFiles/main.js

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
Please fix indentation issues. Whole file is having indentation issues scattered across.